docs/docs/guides/integrate/zitadel-apis/example-zitadel-api-with-dot-net.md

```diff
@@ -43,7 +43,7 @@ dotnet add package Zitadel.Api
 ### Create example client
 
 Change the program.cs file to the content below. This will create a client for the management api and call its `GetMyUsers` function.
-The SDK will make sure you will have access to the API by retrieving a Bearer Token using JWT Profile with the provided scopes (`openid` and `urn:zitadel:iam:org:project:id:{projectID}:aud`).
+The SDK will make sure you will have access to the API by retrieving a Bearer Token using JWT Profile with the provided scopes (`openid` and `urn:zitadel:iam:org:project:id:zitadel:aud`).
```

Comment:
IMO we should also update the text and example to reflect that.
Either we can remove line "The apiProject you will find in the ZITADEL project in the first organization of your instance." or at least rephrase it.
And in the example just exchange
```csharp
const string apiProject = "170078979166961921";
```
with 
```csharp
const string apiProject = "zitadel";
```

Reply:
Ok I missed that, let me look into it